Framework/PythonInterface/mantid/api/src/Exports/SpectrumInfo.cpp

```diff
@@ -14,5 +14,15 @@ void export_SpectrumInfo() {
            "masked.")
       .def("hasDetectors", &SpectrumInfo::hasDetectors, (arg("self")),
            "Returns true if the spectrum is associated with detectors in the "
-           "instrument.");
+           "instrument.")
+      .def("position", &SpectrumInfo::position, (arg("self"), arg("index")),
+           "Returns the position of the detector(s) associated with the "
+           "spectrum.")
```

Comment:
We will probably change the return type of this from `Kernel::V3D` to `Eigen::Vector3d`, so I had not exposed this to Python for now. Probably ok, but worth considering to wait more before making it available.